vss-core/src/main/java/org/eclipse/kuksa/vsscore/annotation/VssDefinition.kt

```diff
@@ -17,23 +17,15 @@
  *
  */
 
-package org.eclipse.kuksa.testapp.util
-
-import kotlin.reflect.KClass
+package org.eclipse.kuksa.vsscore.annotation
 
 /**
- * Utility class used to provide LogTags for Android Logger.
+ * Add this annotation to any class to trigger the model generation (Kotlin Symbol Processing) for the given
+ * Vehicle Signal Specification definition file. Only .yaml files are currently supported. The searched root folder
+ * is the assets folder.
```

Comment:
give an example path here here. examples are easier to understand, replicate and less error prone :)